cli/pkg/config/cli_config_test.go

```diff
@@ -60,6 +64,12 @@ func TestStoreCLIConfig(t *testing.T) {
 	}
 
 	data, err := fileutils.ReadFileAsStr(mng.CLIConfigPath)
+
+	// This is to remove the indentation viper adds
+	data = strings.ReplaceAll(data, " ", "")
```

Comment:
Viper stores the config like this
```json
{
  "automatic_version_check":true,
  "current-context":"",
  "kube_context_check":true,
  "last_version_check":"2020-02-20T00:00:00Z"
}
```
But we want to flatten it out (remove indentations) so that we can compare it with the `testConfig`